components/match2/wikis/trackmania/match_summary.lua

```diff
@@ -0,0 +1,382 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local VodLink = require('Module:VodLink')
+local Json = require('Module:Json')
+local Abbreviation = require('Module:Abbreviation')
+local String = require('Module:StringUtils')
+local Flags = require('Module:Flags')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper', {requireDevIfEnabled = true})
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+local OpponentDisplayCustom = Lua.import('Module:OpponentDisplay/Custom', {requireDevIfEnabled = true})
```

Comment:
instead of using OpponentDisplay and OpponentDisplayCustom just use `require('Module:OpponentLibraries').OpponentDisplay'